### PR TITLE
Add PolymerElement.prototype.updateStyles

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: polymer.externs.js [hunk 1]
```diff
@@ -425,6 +425,21 @@ PolymerElement.prototype.arrayDelete = function(array, item) {};
...
 /**
+ * Re-evaluates and applies custom CSS properties based on dynamic
+ * changes to this element's scope, such as adding or removing classes
+ * in this element's local DOM.
+ *
+ * For performance reasons, Polymer's custom CSS property shim relies
+ * on this explicit signal from the user to indicate when changes have
+ * been made that affect the values of custom properties.
+ *
+ * @param {Object=} properties Properties object which is mixed into
```
Comment (jklein24) [inline polymer.externs.js]: When is this object undefined? When is it null?

File: polymer.externs.js [hunk 2]
```diff
@@ -425,6 +425,21 @@ PolymerElement.prototype.arrayDelete = function(array, item) {};
...
+ *   the element's `customStyle` property. This argument provides a shortcut
```
Comment (jklein24) [inline polymer.externs.js]: nit: 4sp indent here and next line.

Comment (mrdewitt) [inline polymer.externs.js]: Note that the file has a mix of 2sp and 4sp indentation.  I recommend running a linter tool on it.

File: polymer.externs.js (hunk 1 above)
Comment (rictic) [inline polymer.externs.js]: Done.

File: polymer.externs.js (hunk 2 above)
Comment (rictic) [inline polymer.externs.js]: Updated to 4sp, punting on jsdoc linter for now.

**Comment by jklein24** [comment]
LGTM

**Pull request merged**